test/conf/docker-compose.yml

```diff
@@ -0,0 +1,212 @@
+version: '3.5'
+
+x-definitions: &env
+    # IMPORTANT!!
+    # Change this to whatever you want your garden name / namespace default to be
+
+    # If you are running in Ubuntu, then you need to set the Timezone
+    # We are actively investigating why this is
+    TZ: UTC
+
+    # IMPORTANT!!
+    # Change this to support both local and remote plugins
+    # Should be resolvable from inside and outside the docker network
+    BG_PLUGIN_MQ_HOST: rabbitmq
+
+    # IMPORTANT!!
+    # If you are connection to a Parent BG instance, toggle true
+
+
+    # IMPORTANT !!
+    # If you are connecting to a Parent BG instance, change this the resolvable DNS entry
+
+
+
+services:
+    beer-garden:
+        image: bgio/beer-garden:unstable
+        networks:
+            - bg-network
+        ports:
+            - "2337:2337"
+            - "2338:2338"
+        volumes:
+            - ./data/logging-config.yaml:/src/example_configs/logging-config.yaml
+            - ./data/localplugins:/plugins
+        environment:
+            <<: *env
+
+            BG_GARDEN_NAME: docker
+            BG_PARENT_HTTP_ENABLE: "false"
+            BG_PARENT_HTTP_HOST: localhost
+
+            # Point at the correct database
+            BG_DB_HOST: mongodb
+            BG_DB_NAME: beer_garden_v3
+
+            # Point at the correct rabbit broker
+            BG_MQ_HOST: rabbitmq
+            BG_MQ_EXCHANGE: beer_garden
+            BG_MQ_CONNECTIONS_ADMIN_USER: beer_garden
+            BG_MQ_CONNECTIONS_ADMIN_PASSWORD: password
+            BG_MQ_CONNECTIONS_MESSAGE_USER: beer_garden
+            BG_MQ_CONNECTIONS_MESSAGE_PASSWORD: password
+
+            # Enable stomp and point at the correct broker
+            BG_ENTRY_STOMP_ENABLED: "false"
+            BG_ENTRY_STOMP_HOST: activemq
+
+            # We expose grafana for you, but you can change this to an
+            # external grafana URL if you would like. It just adds a
+            # link in our About section if it is defined.
+            BG_METRICS_URL: http://localhost:3000
+            BG_METRICS_PROMETHEUS_ENABLED: "false"
+        depends_on:
+            - mongodb
+            - rabbitmq
+
+    ui:
+        image: bgio/ui:3
+        networks:
+            - bg-network
+        ports:
+            - "80:80"
+              #- "443:443"
+        environment:
+            BEERGARDEN_HOST: "beer-garden"
+            BEERGARDEN_PORT: "2337"
+        depends_on:
+            - beer-garden
+
+    grafana:
+        image: grafana/grafana:5.2.3
+        ports:
+            - "3000:3000"
+        volumes:
+            - ./data/grafana/provisioning:/etc/grafana/provisioning
+            - ./data/grafana/dashboards:/var/lib/grafana/dashboards
+        networks:
+            - bg-network
+        depends_on:
+            - prometheus
```

Comment:
Do we need prometheus and grafana? If not can we remove them? Less moving parts in the test is always good.

Reply:
Yeah, I can remove them.